src/main/kotlin/api/EventsApi.kt

```diff
@@ -21,7 +21,7 @@ interface EventsApi {
     suspend fun triggerEvent(@Body body: TriggerEventRequest): Response<ResponseWrapper<TriggerResponse>>
 
     @POST("$ENDPOINT/trigger/bulk")
-    suspend fun bulkTriggerEvent(@Body body: BulkTriggerEventRequest): Response<ResponseWrapper<TriggerResponse>>
+    suspend fun bulkTriggerEvent(@Body body: BulkTriggerEventRequest): Response<ResponseWrapper<List<TriggerResponse>>>
```

Comment:
Looks good but you need to modify the unit test based on your changes

Reply:
done

Reply:
LGTM @therealansh @unicodeveloper @Cliftonz